src/BasketBundle/Controller/BasketController.php

```diff
@@ -35,6 +35,19 @@
  */
 class BasketController extends Controller
 {
+    /**
+     * We inject alias 'sonata.basket.factory'.
+     */
+    private $basketFactory;
+
+    /**
+     * @param type $basketFactory
+     */
+    public function __construct($basketFactory)
```

Comment:
Please use a typehint here

Reply:
i've remove entirely as @greg0ire proposed

Reply:
I proposed removing the phpdoc. Please use a typehint

Reply:
please confirm it's ok now ?

Reply:
It is :)